.spelling

```diff
@@ -124,9 +136,11 @@ NoSQL
 npm
 NSIS
 NumPy
+oauth
 ORM-like
 optout
 parquet2hive
+passcode
```

Comment:
Appears to be unused.

Reply:
used once in `book/datasets/fxa_metrics/funnels.md` but ill add a space and remove the exception